docs/grafana/README.md

```diff
@@ -30,7 +30,7 @@ grafana-port: 3000        # port for grafana
 
 # Deployment
 
-The deployment of web portal goes with the bootstrapping process of the whole PAI cluster, which is described in detail in [Tutorial: Booting up the cluster](https://github.com/Microsoft/pai/blob/master/pai-management/doc/cluster-bootup.md).
+The deployment of web portal goes with the bootstrapping process of the whole PAI cluster, which is described in detail in [Tutorial: Booting up the cluster](https://github.com/Microsoft/pai/blob/master/docs/pai-management/doc/cluster-bootup.md).
```

Comment:
What about using relative path? So it won't break on different branch.

Reply:
fixed to relative